modules/services/window-managers/i3-sway/lib/functions.nix

```diff
@@ -155,4 +155,6 @@ rec {
     "for_window ${criteriaStr criteria} floating enable";
   windowCommandsStr = { command, criteria, ... }:
     "for_window ${criteriaStr criteria} ${command}";
+  workspaceOutputStr = item:
+    ''workspace "${item.workspace}" output ${item.output}'';
```

Comment:
```suggestion
    "workspace "${item.workspace}" output ${item.output}";
```

Reply:
Seems to be not possible as ./format is changing it back when I change to double-quotes and escape the needed double-quotes around `${item.workspace}`.